action.py

```diff
@@ -278,18 +278,15 @@ def _fatal_help(msg):
 _github_env = os.getenv("GITHUB_ENV")
 assert _github_env is not None
 with Path(_github_env).open("a") as gh_env:
+    delim = os.urandom(16).hex()
```

Comment:
What's the idea behind this?

Reply:
I took that from GitHub's docs on best practices for action files: apparently it's considered best practice to use a random delimiter, to avoid any possibility of a collision with the input.

Reply:
Ah ok, can you leave a comment to that effect? Otherwise, it looks good.